### PR TITLE
Make setup_default_backend.sh fail fast

### PR DESCRIPTION
**Pull request by raphj**
Related to https://github.com/tracim/tracim/issues/2979

This pull request:
 - groups the apt install commands
 - makes the error stop the script with a return value of 1.

I looked up closed tickets to know if this would cause an installation fail that was passing with an harmless error, I didn't find anything with the keywords apt, install, package, setup, setup_default_backend.

## Checkpoints

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

**For developer**

- [x] Manual tests done to ensure stability on whole application layers, issue expectation follow
- [x] Original issue have been updated with a short resume of implemented solution

**For quality team**

- [x] Tested by quality team

### CODE REVIEW

**Comment by raphj** [comment]
I checked "Automated tests" because it does not seem relevant here

File: setup_default_backend.sh [hunk 1]
```diff
@@ -36,10 +37,7 @@ function install_backend_system_dep {
     log "install base debian-packaged-dep for backend..."
     $SUDO apt update
     PACKAGE_LIST='python3 python3-venv python3-dev python3-pip redis-server zlib1g-dev libjpeg-dev imagemagick libmagickwand-dev libpq-dev ghostscript libfile-mimeinfo-perl poppler-utils libimage-exiftool-perl qpdf libldap2-dev libsasl2-dev libreoffice inkscape ufraw-batch ffmpeg'
-    for PACKAGE in $PACKAGE_LIST
-    do
-        $SUDO apt install -y $PACKAGE && loggood "$PACKAGE correctly installed" || logerror "failed to install $PACKAGE"
-    done
```
Comment (PhilippeAccorsi) [inline setup_default_backend.sh:42]: I remember why we have used this. On some installation we have seen packages error and its not easy to find which packages is not installed correctly in all log.

Comment (raphj) [reply setup_default_backend.sh:42]: apt should print which packages are problematic, no? Do you think this is still problematic?

Comment (PhilippeAccorsi) [reply setup_default_backend.sh:42]: I need to search in history of this fix to see if an issue exist (and explain why we have used this).

Comment (inkhey) [reply setup_default_backend.sh:42]: Did you get any information on this ?
If not, we may put this in 3.2.X and solve this later.

Comment (raphj) [reply setup_default_backend.sh:42]: after discussion, Phillippe agrees with this design

**Pull request merged**